packages/server/tests/acceptance/rpc_batch1.spec.ts

```diff
@@ -718,7 +719,9 @@ describe('@api-batch-1 RPC Server Acceptance Tests', function () {
           requestId,
         );
 
-        expect(receiptFromRelay.effectiveGasPrice).to.equal(prepend0x(currentPrice.toString(16)));
+        // handle deviation of 20% in gas price
+        expect(receiptFromRelay.effectiveGasPrice).to.be.lessThan(Number(prepend0x(currentPrice)) * 1.2);
```

Comment:
Instead of 20% with hardcoded 1.2 and 0.8 why not add a test config that allows us to set the % deviation as a decimal

Reply:
The effectiveGasContract.spec.ts has the % deviation hardcoded.  I'll make the update.